src/EditorFeatures/Core/StackTraceExplorer/StackTraceExplorerUtilities.cs

```diff
@@ -0,0 +1,211 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.FindUsages;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.StackFrame;
+using Microsoft.CodeAnalysis.FindUsages;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.StackTraceExplorer;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.StackTraceExplorer
+{
+    internal static class StackTraceExplorerUtilities
+    {
+        public static async Task<DefinitionItem?> GetDefinitionAsync(Solution solution, StackFrameCompilationUnit compilationUnit, StackFrameSymbolPart symbolPart, CancellationToken cancellationToken)
+        {
+            // MemberAccessExpression is [Expression].[Identifier], and Identifier is the 
+            // method name.
+            var typeExpression = compilationUnit.MethodDeclaration.MemberAccessExpression.Left;
+            var fullyQualifiedTypeName = typeExpression.ToString();
```

Comment:
does this properly handle type argument arity?  if so, mentino that.

Reply:
It does. I'll add a comment